docs/user-guide/integrations.md

```diff
@@ -51,3 +51,7 @@
 * [React](https://github.com/yannickcr/eslint-plugin-react)
 
 … and many more published on npm with the [eslintplugin](https://www.npmjs.com/browse/keyword/eslintplugin) keyword.
+
+## Awesome ESLint
```

Comment:
I'd maybe have this header be a bit more generic, as it's not obvious what this means. Maybe "Other Integration Lists"?